dll/win32/uxtheme/errinfo.c

```diff
@@ -0,0 +1,163 @@
+/*
+ * PROJECT:     ReactOS uxtheme.dll
+ * LICENSE:     LGPL-2.1-or-later (https://spdx.org/licenses/LGPL-2.1-or-later)
+ * PURPOSE:     Error information of UXTHEME
+ * COPYRIGHT:   Copyright 2025 Katayama Hirofumi MZ <katayama.hirofumi.mz@gmail.com>
+ */
+
+#include "uxthemep.h"
+#include <stdlib.h>
+#include <strsafe.h>
+
+HRESULT
+UXTHEME_MakeError32(_In_ LONG error)
```

Comment:
It looks like this function (and UXTHEME_MakeLastError) could be kept in the header and made inline.

Reply:
Moved to header and make it inline.